src/main/java/com/playmonumenta/networkrelay/RemotePlayerManagerPaper.java

```diff
@@ -189,7 +210,7 @@ public void playerCommandPreprocessEvent(PlayerCommandPreprocessEvent event) {
 		if (command.startsWith("/pv ")
 			|| "/pv".equals(command)
 			|| command.contains("vanish")) {
-			Bukkit.getScheduler().runTask(NetworkRelay.getInstance(), this::refreshLocalPlayers);
+			Bukkit.getScheduler().runTask(NetworkRelay.getInstance(), () -> refreshLocalPlayers());
```

Comment:
Is there any reason for this change? IntelliJ prefers method references over lambdas that call a single method.

Reply:
The compiler said it was too ambiguous